mail/header.go

```diff
@@ -308,6 +364,35 @@ func (h *Header) MsgIDList(key string) ([]string, error) {
 	return l, nil
 }
 
+// MsgIDList parses a list of URLs from a list command header. It returns URLs.
+// If the header field is missing, it returns nil.
+//
+// This can be used on List-Help, List-Unsubscribe, List-Subscribe, List-Post,
+// List-Owner, and List-Archive headers.
+//
+// See https://www.rfc-editor.org/rfc/rfc2369 for more information.
```

Comment:
Nit: we can just use plaintext to reference the RFC: "See RFC 2369". GoDoc will automatically linkify it, and it's more readable.